homework5/task2.py

```diff
@@ -0,0 +1,4 @@
+list1 = ['Hillel', 'AQA', 'TEST']
+string_from_list1 = str(list1)
+list2 = [string_from_list1]
+print(list2)
```

Comment:
Invalid decision. You should use method .join()

Reply:
I have fixed it.